test/functional/finalization_deposit_reorg.py

```diff
@@ -0,0 +1,218 @@
+#!/usr/bin/env python3
+# Copyright (c) 2019 The Unit-e developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
```

Comment:
Missing a description of what this test is aiming to test

Reply:
Added a sentence